pkg/injector/patch.go

```diff
@@ -38,24 +37,71 @@ func (wh *mutatingWebhook) createPatch(pod *corev1.Pod, req *admissionv1.Admissi
 	metricsstore.DefaultMetricsStore.CertIssuedCount.Inc()
 	metricsstore.DefaultMetricsStore.CertIssuedTime.
 		WithLabelValues().Observe(elapsed.Seconds())
-	originalHealthProbes := rewriteHealthProbes(pod)
+	originalHealthProbes := rewriteHealthProbes(pod) // need to get these out of here...
```

Comment:
Please undo this comment or make it more explicit, as I find it vague and confusing.

Reply:
done!